tenseal/cpp/tensors/ckksvector.h

```diff
@@ -122,6 +122,13 @@ class CKKSVector {
     CKKSVector polyval(const vector<double>& coefficients);
     CKKSVector& polyval_inplace(const vector<double>& coefficients);
 
+    /*
+    Image Block to Columns .
+    */
+    CKKSVector conv2d_im2col(const vector<double>& kernel, size_t windows_nb);
+    CKKSVector& conv2d_im2col_inplace(const vector<double>& kernel,
+                                      size_t windows_nb);
+
```

Comment:
I think it's more like a convolution with a special input, can you specify convolution and how the input should be encoded?

Reply:
@philomath213 :point_up: